### DISCUSSION TITLE
ModuleNotFoundError: No module named 'packaging' ERROR

### DISCUSSION THREAD

**Discussion by patrikgyllhamn** [TomSchimansky/CustomTkinter#2179] [General]
Hi!

I keep getting this error in my Virtual Studio Code console when I for exampel run one of your examples of CTK. 

ModuleNotFoundError: No module named 'packaging'

Iv installed and updated CTK 

I use windows and Im a beginner. 

Thanksfull for help.

**Comment by dipeshSam** [comment]
@patrikgyllhamn Try to install packaging module using the following command:
```pip install packaging ```

If still issue persists, try to check environment setup for Python.
